pydeseq2/dds.py

```diff
@@ -238,6 +238,21 @@ def deseq2(self):
             # for genes that had outliers replaced
             self.refit()
 
+    @property
+    def normed_counts(self):
+        """
+        Return median-of-ratios normalized counts.
+
+        Called like an attribute (``dds.normed_counts``) thanks to the
+        ``@property`` decorator.
+        """
+
+        # Check that size factors are available. If not, compute them.
+        if not hasattr(self, "size_factors"):
+            self.fit_size_factors()
+
+        return self.counts.div(self.size_factors, 0)
```

Comment:
```suggestion
        return self.counts.div(self.size_factors, axis=0)
```

As `axis` is an optional argument of the [`div` method](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.div.html)?